pom.xml

```diff
@@ -1,16 +1,16 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.jgroups.aws.s3</groupId>
-    <artifactId>native-s3-ping</artifactId>
+    <groupId>org.jgroups.aws</groupId>
+    <artifactId>jgroups-aws</artifactId>
     <packaging>jar</packaging>
-    <name>native-s3-ping</name>
+    <name>JGroups - AWS</name>
     <version>2.0.1.Final-SNAPSHOT</version>
```

Comment:
Since we are changing the repository name, maven coordinates, and protocol name, I would suggest incrementing the major version to 3.

Reply:
Ideally we would also use Maven relocation

Reply:
@tristantarrant Yes, we could – but that's done in the "old branch" not this branch (`main`).

Reply:
Opened https://github.com/jgroups-extras/jgroups-aws/issues/172 issue to track relocation pom.